Sources/AppcuesKit/Presentation/Traits/Appcues/ExperienceWrapperViewController.swift

```diff
@@ -70,18 +69,67 @@ internal class ExperienceWrapperViewController<BodyView: ExperienceWrapperView>:
 
         bodyView.setNeedsLayout()
 
+        configureConstraints(style)
+
+        switch transition {
+        case .fade:
+            modalTransitionStyle = .crossDissolve
+            modalPresentationStyle = .overFullScreen
+        case let .slide(edge):
+            modalPresentationStyle = .custom
+            transitioningDelegate = self
+            let animator = ExperienceWrapperSlideAnimator(view: bodyView, edge: edge)
+            slideAnimationController = animator
+        }
+
+        return self
+    }
+
+    private func configureConstraints(_ style: ExperienceComponent.Style?) {
+        let contentView = bodyView.contentWrapperView
+
         switch style?.verticalAlignment {
         case "top":
-            bodyView.contentWrapperView.topAnchor.constraint(equalTo: bodyView.safeAreaLayoutGuide.topAnchor).isActive = true
+            contentView.topAnchor.constraint(
+                equalTo: bodyView.safeAreaLayoutGuide.topAnchor,
+                constant: style?.marginTop ?? 0
+            ).isActive = true
         case "bottom":
-            let constraint = bodyView.contentWrapperView.bottomAnchor.constraint(equalTo: bodyView.safeAreaLayoutGuide.bottomAnchor)
+            let constraint = contentView.bottomAnchor.constraint(
+                equalTo: bodyView.safeAreaLayoutGuide.bottomAnchor,
+                constant: -1 * (style?.marginBottom ?? 0)
+            )
             constraint.isActive = true
             bottomConstraint = constraint
         default:
-            bodyView.contentWrapperView.centerYAnchor.constraint(equalTo: bodyView.centerYAnchor).isActive = true
+            contentView.centerYAnchor.constraint(equalTo: bodyView.centerYAnchor).isActive = true
         }
 
-        return self
+        switch style?.horizontalAlignment {
+        case "leading":
+            contentView.leadingAnchor.constraint(
+                equalTo: bodyView.layoutMarginsGuide.leadingAnchor,
+                constant: style?.marginLeading ?? 0
+            ).isActive = true
+        case "trailing":
+            contentView.trailingAnchor.constraint(
+                equalTo: bodyView.layoutMarginsGuide.trailingAnchor,
+                constant: -1 * (style?.marginTrailing ?? 0)
+            ).isActive = true
+        default:
+            contentView.centerXAnchor.constraint(equalTo: bodyView.centerXAnchor).isActive = true
+        }
+
+        if let width = style?.width, width > 0 {
+            let widthConstraint = contentView.widthAnchor.constraint(equalToConstant: width)
+            // lower priority here so that the constraint on width being >= the readableContentGuide width
```

Comment:
`>= the readableContentGuide width` or `<=`?

Reply:
oops, fixed.